### DISCUSSION TITLE
Observation about correlated main window and submenu movement

### DISCUSSION THREAD

**Discussion by mcbeaker** [scanberg/viamd#44] [Q&A] [answered]
When I open the main window full screen, adjust the location of the submenus by spreading them out across the main window, and then decrease the width of the main window, the sub windows get pushed towards the side of the main window that I am reducing the size towards, when I expand the window the submenus are now on top of each other. Is this expected behavior?

**Comment by mathieulinares** [comment]
Thanks for the feedback. This is not an expected behavior and might have to do with your specific setup.
May I ask if you kept the standard cmake option of if you have changed the IMGUI option to enable viewports? Because this option is very buggy. However we are planning to update Imgui version very quick. It might improve this behavior. I'll update this Q&A when done.

**Reply by mcbeaker** [reply]
I kept the standard cmake without any additional parms. Thank you, additionally when I adjusted the height of the window the protein moved away, while the submenus did not move. Whereas if I adjust the width the protein doesn't move in the z-axis, however it does move along the x/y axis

**Reply by mathieulinares** [reply]
We have updated Imgui. Could you pull, recompile and try again and tell us if you experience the same issue!?
Best.
Mathieu

**Reply by mcbeaker** [reply]
Thank you @mathieulinares for updating Imgui. I first tried to pull, and recompile and I ran into a problem. I did not think to log the error that I got. But I think it was because I was in a conda environment. However, after removing the viamd/ directory and cloning the project and submodules, I was able to run viamd and the submenus did not change location when I changed the size of the main window! Woohoo - much thanks!

**Reply by mathieulinares** [reply, marked as answer]
Glad to hear that @mcbeaker. I will mark that q&a as answered. Cheers
